### PR TITLE
update README, Containerfile

### PR DESCRIPTION
**Pull request by sallyom**
@hemajv This PR uses the static ffmpeg from https://hub.docker.com/r/mwader/static-ffmpeg/ and uses multi-stage Containerfile to keep the image small. Also, added the default AUDIO_FILE to be the `jfk.wav` from the whisper.cpp repo. 

This seems to work, can you please build it and try it out? If it looks good, you can please merge this into your branch?

### CODE REVIEW

**Comment by hemajv** [comment]
@sallyom thank you so much for looking into this! will give this a try 👍

File: whisper-playground/README.md [hunk 1]
```diff
@@ -21,21 +21,35 @@ cd ../
...
+Whisper.cpp requires as an input 16-bit WAV audio files.
+By default, a sample `jfk.wav` file is included in the whisper image. This can be used to test with.
+To convert your input audio files to 16-bit WAV format you can use `ffmpeg` like this:
 
 ```bash
 ffmpeg -i <input.mp3> -ar 16000 -ac 1 -c:a pcm_s16le <output.wav>
 ```
 
-Make sure to download your audio files in your `Local/path/to/locallm/models` folder.
+The environment variable `AUDIO_FILE` can be passed to override the default `/app/jfk.wav` file within the whisper image.
```
Comment (hemajv) [inline whisper-playground/README.md]: Can we update this to: `The environment variable 'AUDIO_FILE', can be passed with your own audio file to override the default '/app/jfk.wav' file within the whisper image.`

File: whisper-playground/README.md [hunk 2]
```diff
@@ -21,21 +21,35 @@ cd ../
...
 
 ### Deploy Model
 
 Deploy the LLM and volume mount the model of choice.
+Here we are mounting the `ggml-small.bin` modedl downloaded above.
```
Comment (hemajv) [inline whisper-playground/README.md]: Small nit fix, it should be: `Here, we are mounting the 'ggml-small.bin' model as downloaded from above.`

File: whisper-playground/README.md [hunk 3]
```diff
@@ -21,21 +21,35 @@ cd ../
...
+
+To test with the included `/app/jfk.wav` audio file:
```
Comment (hemajv) [inline whisper-playground/README.md]: small nit, can we update this to: `To test with the default '/app/jfk.wav' audio file included in the image:`

**Review by hemajv — CHANGES_REQUESTED** [review]
Tested this out and it works 🎉
left a few comments for minor changes, but other than that it looks great!
thanks @sallyom 😄

File: whisper-playground/README.md (hunk 1 above)
Comment (sallyom) [reply whisper-playground/README.md]: done

File: whisper-playground/README.md (hunk 2 above)
Comment (sallyom) [reply whisper-playground/README.md]: done

File: whisper-playground/README.md (hunk 3 above)
Comment (sallyom) [reply whisper-playground/README.md]: done

**Review by hemajv — APPROVED** [review]

**Pull request merged**